symengine/cwrapper.cpp

```diff
@@ -399,9 +399,17 @@ void basic_pow(basic s, const basic a, const basic b)
     s->m = SymEngine::pow(a->m, b->m);
 }
 
-void basic_div(basic s, const basic a, const basic b)
+int basic_div(basic s, const basic a, const basic b)
 {
-    s->m = SymEngine::div(a->m, b->m);
+    try
+    {
+        s->m = SymEngine::div(a->m, b->m);
+        return 0;
+    }
+    catch ( ... )
```

Comment:
got it.. generally what are the other errors you'd like to have specialized in the SymEngine?

I'm thinking of `ParseError` for parsing and `MatrixError` for matrices.

So, the general structure of exception handling is okay.. right? The parts in the Ruby wrappers as well? If so I can go ahead and include similar exception handling code for all the wrapped functions. (Except for those which are expected to return some values, thinking of the best way to handle them now)